apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,57 @@
 import { Component, OnInit } from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
+import { City } from '../../model/city.model';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { CityStore } from '../../data-access/city.store';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      [imgUrl]="'assets/img/city.webp'"
```

Comment:
You could just pass in the whole `<img>` tag along with an identifier as content beneath this that you can reference with another `<ng-content>` tag inside of card.component.html.